docs/Azure_IoT_Hub_integration/Concepts/LwM2M_mappings.md

```diff
@@ -0,0 +1,61 @@
+# LwM2M mappings
+
+In this section you'll get to know how the mappings are arranged between the LwM2M protocol as used in Coiote DM and the data retrieval and processing mechanisms of the Azure IoT Hub, such as Device Twins, Direct Method and message routing interface (Device-to-cloud messaging).
+
+## Introduction
+
+The LwM2M protocol data model is organized as a three-level tree that has the following structure:
+
+ - **object** (e.g. a 'temperature sensor')
+    - **object instance** (e.g. 'temperature sensor #1', 'temperature sensor #2' etc.)
+        - **resource** (e.g. 'current temperature value')
+
+In terms of operations that can be performed on an LwM2M Client, an LwM2M Server can READ all of the data model entities, and, depending on their characteristics, may also WRITE to some of them, and EXECUTE some of them. Additionally, an LwM2M Server can also OBSERVE selected resources.
+
+This division into readable, writable, executable and observable data model entities is the basis for the mapping of LwM2M resources (as interpreted by Coiote DM) into Azure IoT Hub data processing mechanisms.        
+
+## LwM2M readable and writable resources
+
+Within the Coiote DM - Azure IoT Hub integration, readable and writable resources are usually interpreted as part of Azure Device twin data structure.
+
+!!! note
+    To learn more about Device twins, go to [Understand and use Device twins](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins) section of the Azure IoT Hub documentation.
+
+### READ and WRITE - Communication flow
+
+This means that standard data model resources, such as `Manufacturer` (with ID **3/0/0**) will be mapped into the Device twin as a reported property.
+
+![Device twin READ](images/Device_twin_read.png "READ Manufacturer")
+
+On the other hand, a writable resource, such as `Lifetime` (with ID **1/0/1**) will be also mapped as a desired property, enabling you to synchronize the device data model and configuration between Azure and Coiote DM.
```

Comment:
Unfortunately, it doesn't work like that. The writable resource is also mapped as reported property. 
To change the value of that resource the user needs to create proper json in desired property field (in Device Twins json) and click save. Then Azure sends the change to Coiote and then to the device. And when coiote handle that the value has changed on the device it reports to Azure that the value of the reported part of device twins should be updated to the new value. Is this clear enaugh?

Reply:
yup, changed accordingly